### PR TITLE
add condition to s3 bucket

### PR DESCRIPTION
**Pull request by ismaelfernandezscmspain**
@fiunchinho @aramirez-es @Hyunk3l 
- I have added conditional for logging elb

Can you review it?

### CODE REVIEW

File: troposphere/asg-elb-dns.py [hunk 1]
```diff
@@ -105,6 +105,8 @@ def generate_cloudformation_template():
             Description="S3 Bucket for the ELB access logs"
         ))
 
+        template.add_condition("LogginELBCondition", Not(Equals(Ref(elb_bucket_name), "")))
```
Comment (Hyunk3l) [inline troposphere/asg-elb-dns.py]: `LogginELBCondition` is correct?

Comment (ismaelfernandezscmspain) [reply troposphere/asg-elb-dns.py]: @Hyunk3l LoggerToELB ?

Comment (Hyunk3l) [reply troposphere/asg-elb-dns.py]: `ElbLoggingCondition`
`ELBLoggingCondition`
or something similar imo

**Review by fiunchinho — APPROVED** [review]

File: troposphere/asg-elb-dns.py (hunk 1 above)
Comment (ismaelfernandezscmspain) [reply troposphere/asg-elb-dns.py]: @Hyunk3l your wishes are order to me

**Review by Hyunk3l — APPROVED** [review]

**Pull request merged**